### ISSUE TITLE
Just so you know

### ISSUE THREAD

**Issue by EduardoJimenez** [codediodeio/geofirex#122]
Please include in the documentation a warning that compound **queries with orderby and range wheres** do not work with this package, 

This is not a problem, per se, of this package but of how queries are limited by firestore.

The thing is that it fails **silently** and you come to suspect that is a problem with GeoFirex, but again, it is not.

**Comment by josephchang** [comment]
Yeah, I was trying to make a query within an X radius and filtered by a date range, but I was unable to do so.

**Comment by devnoot** [comment]
Ran into the same issue. Can you shed some light on why this limitation occurs?

**Comment by samarthmn** [comment]
Any workaround?
